currency-rates/internal/repo/subsctiption.go

```diff
@@ -8,19 +8,25 @@ import (
 
 type DBEmail struct {
     Email string
+	IsActive bool
 }
 
 func (repo *Repository) SetEmail(ctx context.Context, email string) error {
 	e := &DBEmail{
 		Email: email,
+		IsActive: true,
 	}
 
 	return repo.DB.Table("emails").Create(e).Error
 }
 
 func (repo *Repository) GetEmails(ctx context.Context) ([]services.Email, error) {
 	result := make([]services.Email, 0)
-	err := repo.DB.Table("emails").Find(&result).Error
+	err := repo.DB.Table("emails").Where("is_active = ?", true).Find(&result).Error
```

Comment:
So maybe your method should be renamed to 'GetAllActiveEmails'?

Reply:
Done